otaigbe/lendMe
# Build out the get loans endpoint

#### What does this PR do?
Build out the get available loans endpoint
#### Description of Task to be completed?
Have the following endpoint working
- POST /api/v1/loans
#### How should this be manually tested?
- After cloning the repo, cd into it and run the command 'npm start'
- Open Postman
- signin/signup see(https://github.com/otaigbe/lendMe/pull/1)
- type in the endpoint url
- select the get method
- add the 'x-auth-token' key and the token gotten from login
- click send
